web-components/src/components/image-grid/index.tsx

```diff
@@ -121,12 +106,21 @@ export default function ImageGrid({
         <Title variant="h2" className={classes.title}>
           {title}
         </Title>
-        <Description
-          className={classes.description}
-          fontSize={getFontSize('xl')}
+        <Body
+          size="xl"
+          sx={theme => ({
+            ml: even ? 'auto' : 0,
+            mr: even ? 0 : 'auto',
+            [theme.breakpoints.between('md', 'xl')]: {
+              maxWidth: theme.spacing(115.5),
+            },
+            [theme.breakpoints.up('xl')]: {
+              maxWidth: theme.spacing(145),
+            },
```

Comment:
I don't think we should get in the habit of using this version of responsive styles (and should instead favor the more simple `sx` syntax, however there are several spots like this I do so just so i could copy over the prior styles